recipes/local.php

```diff
@@ -94,6 +94,8 @@
     $branch = env('branch');
     $gitCache = env('local_git_cache');
     $depth = $gitCache ? '' : '--depth 1';
+    $gitCloneTimeout = env('local_git_clone_timeout');
+    $gitCloneTimeout = $gitCloneTimeout ? $gitCloneTimeout : 60;
```

Comment:
You should look at how it's done in rsync recipe. https://github.com/deployphp/recipes/blob/4f21808a3278855ed1e3afdebbf1d48afeb831f2/recipes/rsync.php#L21